lib/apis/archive-files.js

```diff
@@ -141,10 +142,36 @@ module.exports = class ArchiveFilesAPI {
 
     // handle folder
     if (entry && entry.isDirectory()) {
-      res.writeHead(200, 'OK', {
-        'Content-Type': 'text/html'
+      var negotiator = new Negotiator(req)
```

Comment:
What about https://expressjs.com/en/4x/api.html#req.accepts or https://expressjs.com/en/4x/api.html#res.format ? Do they mishandle accept headers?

Reply:
I wasn't aware of those, they would probably work fine.